bin/prettier.js

```diff
@@ -145,7 +146,10 @@ function getTrailingComma() {
   }
 }
 
+const r = require;
 const options = {
+  astTransform: argv["ast-transform"] &&
```

Comment:
How about calling it `transform`? It's mostly an implementation detail that transforms operate on the AST.